deeppavlov/configs/squad/squad_zh_bert_mult.json

```diff
@@ -52,6 +52,16 @@
         "class_name": "squad_bert_ans_postprocessor",
         "in": ["ans_start_predicted", "ans_end_predicted", "context_raw", "bert_features", "subtok2chars"],
         "out": ["ans_predicted", "ans_start_predicted", "ans_end_predicted"]
+      },
+      {
+        "in": "ans",
+        "out": "ans_tok",
+        "class_name": "jieba_tokenizer"
+      },
+      {
+        "in": "ans_predicted",
+        "out": "ans_predicted_tok",
+        "class_name": "jieba_tokenizer"
       }
```

Comment:
If this component modifies answers to properly calculate metrics, shouldn't they be a part of the metrics themselves?